.github/workflows/tests.yaml

```diff
@@ -9,48 +9,49 @@ on:
     branches:
       - main
 
+# ensuring only one instance is running at a given time
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   test-conda-store-server:
     name: "unit-test conda-store-server"
     strategy:
       matrix:
         # cannot run on windows due to needing fake-chroot for conda-docker
-        os: [ubuntu-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
+        os: ["ubuntu", "macos"]
+        include:
+          - os: ubuntu
+            environment-file: conda-store-server/environment-dev.yaml
+            miniforge-version: latest
```

Comment:
Will we ever want to run on different versions for different OS's? If not, we could just remove this and add it below since it is currently the same for both envs.

Reply:
I am not sure we will 🤔 I can remove it for now and add back again if needed